src/Workspaces/Core/Portable/CodeStyle/CodeStyleOptions.cs

```diff
@@ -99,5 +99,11 @@ public class CodeStyleOptions
             nameof(PreferInlinedVariableDeclaration),
             defaultValue: TrueWithSuggestionEnforcement,
             storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.PreferInlinedVariableDeclaration"));
+
+        internal static readonly PerLanguageOption<CodeStyleOption<bool>> PreferExplicitTupleNames = new PerLanguageOption<CodeStyleOption<bool>>(
```

Comment:
>PreferExplicitTupleNames [](start = 74, length = 24)

can you file a bug to add this option to editorconfig when that flows into master?